WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeWebViewFragment.java

```diff
@@ -118,6 +121,17 @@ public void onResume() {
         setNavigationIcon(R.drawable.ic_close_white_24dp);
     }
 
+    private void clearCookies() {
```

Comment:
> Am I correct in thinking removeAllCookies() will affect every WebView in the app? If so, this might be a problem since it would force the user to re-login to any authenticated page they visited in the app.

Yup, you're right. I like your emphasis on _might_ because I went through the same thought process. I believe it is ok because I don't see too many use cases where someone has navigated far beyond WordPress-related content. And since with WordPress-related content we are passing the credentials in anyway, they'll always be logged-in.

> Also, keep in mind that clearing cookies is asynchronous, so it's possible that the cookies won't be cleared before the page is browsed. If this is a concern, you may want to flush the cookie manager before browsing (keeping in mind that's a blocking call).

Good call. I'll move it to when the WebView closes (like iOS does)

> One final nit: this seems like it belongs in a utility class since it's not tied to the publicize feature.

That's fair. I'll create (I couldn't find one) a WebViewUtils class.